package.json

```diff
@@ -26,7 +26,7 @@
             "/node_modules/",
             "/lib/",
             "/esm/",
-            "/examples/simple/"
+            "/examples/simplme/"
```

Comment:
Sorry for resurrecting an old PR. I think this change may have not been intentional.